skeleton/controller/acp_controller.php.twig

```diff
@@ -40,16 +42,18 @@ class acp_controller
 	 * Constructor.
 	 *
 	 * @param \phpbb\config\config		$config		Config object
-	 * @param \phpbb\language\language	$lang		Language object
+{% if LANGUAGE.s_32 %}	 * @param \phpbb\language\language	$language	Language object
```

Comment:
Also here lets keep the conditionals on their own lines and if needed use `{%-` or `-%}` to remove extra whitespaces. Same goes for below wherever we're not actually doing in-line edits.